content/docs/pipelines/sdk/output-viewer.md

```diff
@@ -147,14 +147,21 @@ viewers later on the page.
           which case the Kubeflow Pipelines UI concatenates the data from the 
           matching source files.</p>
           
-          <p>For some viewers, this field can contain inlined string 
-          data instead of a path.</p>
+          <p><code>source</code> can also contain inlined string data instead of
+          a path when <code>storage='inline'</code>.</p>
           </td>
       </tr>
       <tr>
         <td><code>storage</code></td>
-        <td>Applies only to outputs of type <code>markdown</code>. See 
-        <a href="#type-markdown">below</a>.</td>
+        <td><p>Optional. When <code>storage</code> is 'inline', value of
+        <code>source</code> is parsed as inline data instead of a path. Applies
+        to all types of outputs except <code>tensorboard</code>. See 
+        <a href="#markdown">markdown type</a> or <a href="#web-app">webapp type</a>
```

Comment:
Do markdown and webapp refer to string values like tensorboard above? If so, please use code font for markdown and webapp. Otherwise, if you are referring to the sections below please change to Markdown and Web app.

Reply:
That's a good point, changed to refer to sections.